src/actions/blog/feed.cr

```diff
@@ -0,0 +1,31 @@
+class Blog::Feed < BrowserAction
+  get "/blog/feed.xml" do
+    posts = PostQuery.new.all[0...10]
+    xml render_feed(posts)
+  end
+
+  private def render_feed(posts)
+    items = posts.map do |post|
+      url = Blog::Show.with(post.slug).url
+      RSS::Item.new(
+        guid: RSS::Guid.new(value: url, is_permalink: true),
+        title: post.title,
+        link: url,
+        description: post.html_content,
+        pub_date: post.published_at,
+      )
+    end
```

Comment:
I'm thinking that some of this could be moved to `RSS:Channel` and that class could be responsible for rendering times, rather than rendering them and passing them in. I'm imagining something like this:

```crystal
RSS::Channel.new(posts: posts).to_xml
```

Then `RSS::Channel` would build the `RSS::Item` with the passed in posts, private method could be added for `last_build_date`, `url`, etc. I also think the title and description of the channel could be hardcoded and not passed in since this is only used for the Lucky Blog. We could make those arguments later if we decide to add more RSS feeds.

Does this make sense? What do you think?